spec/features/logging/view_auth_requests_for_a_username_spec.rb

```diff
@@ -51,13 +70,23 @@
     it "displays the authentication requests" do
       expect(page).to have_content("Displaying logs for #{username}")
     end
+
+    it "displays the logs of the ip the organisation does own" do
+      expect(page).to have_content("1.1.1.1")
+    end
+
+    it "does not display the logs of the ip the organisation does not own" do
+      expect(page).to_not have_content("2.2.2.2")
+    end
   end
 
   context "without results" do
     let(:user) { create(:user, :confirmed, :with_organisation) }
+    let(:admin_user) { create(:user, :confirmed, organisation_id: organisation.id) }
```

Comment:
Do we need both these users?

Reply:
If it's to create a super admin, you should be able to just add `admin: true`
e.g. `let(:admin_user) { create(:user, :confirmed, organisation_id: organisation.id, admin: true) }`
Adding an admin has been pushed into master, so it should work here?